node_modules/.bin/acorn

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
- [ ]  OPTIONAL
Kindly ensure to enlist the _node_modules_ directory  in the `.gitignore`